style.css

```diff
@@ -53,3 +48,9 @@ body {
   padding-top: 10px;
   margin: 0;
 }
+
+@media (max-width: 500px) {
+    .content-box {
```

Comment:
This is not a comment that you need to take action on, but just an FYI.

The CSS `@media` queries can sometimes have problems with specificity, e.g. although we define a rule inside of a `@media` block, something else outside of the block may still override the rule. A nifty work-around for this is to increase the specificity of the CSS rules inside of your `@media` block by doubling the selector. This is a readable way to increase specificity as it can't be mistaken for anything else. Here, you could do: `.content-box.content-box` so that you automatically in the specificity battle against any rule that just selects on `.content-box`.